### ISSUE TITLE
Could not sync file or folder Could not find a suitable TLS CA certificate bundle, invalid path: /Applications/Maestral.app/Contents/Resources/app_packages/certifi/cacert.pem

### ISSUE THREAD

**Issue by tbraun1551** [samschott/maestral#667]
**Describe the bug**

When I check the status of one of my dropbox synced through maestral by running the following cl command: `maestral status -c school`, I get information about the account and something saying the account status is paused as well as the following error text 
```
Could not sync file or folder
Could not find a suitable TLS CA certificate bundle, invalid path:
/Applications/Maestral.app/Contents/Resources/app_packages/certifi/cacert.pem
```

When I run `maestral start -c school` I get a response saying the Daemon is already runnning. I have tried linking and relinking the dropbox account to no avail. I have two dropbox accounts synced through maestral and they both get that error. 

**To Reproduce**

Check status of maestral account through the command line. 

**Expected behaviour**

**System:**

 - Maestral version: 
 - Python version:  Python 3.9.12
 - OS: [e.g. Ubuntu]: MacOS
 - Desktop environment: [e.g. Gnome 3.32]: MacOS
 - PyQt version (for Linux GUI):

**Additional context**

**Comment by samschott** [comment]
@tbraun1551, this is strange, the file `Maestral.app/Contents/Resources/app_packages/certifi/cacert.pem` should indeed be part of the app bundle. Could you check the following?

1. Which version of Maestral are you using? `maestral --version` will give you that info.
2. Where is the location of the app bundle?
3. Do you actually find the above file inside the app bundle?
4. Could you post are more detailed traceback from the logs, accessible via `maestral log show -e`?

**Comment by tbraun1551** [comment]
Increased the log level and quit maestral through activity monitor. I then restarted maestral by running `maestral start -c school` from the CL. This seems to have fixed the issue as I am now able to get both my dropbox accounts working and no longer get the cert error when checking the status. 

Here are the logs just in case they help. Let me know if there is anything else I can provide. 

```
2022-03-02 13:35:52 daemon INFO: Starting daemon
2022-03-02 13:35:52 manager INFO: Connected
2022-03-02 13:35:56 sync INFO: Sync aborted
2022-03-02 13:35:56 manager INFO: Paused
2022-04-07 10:13:07 daemon INFO: Starting daemon
2022-04-07 10:13:07 main INFO: Scheduling reindex after update from pre v1.6.0
2022-04-07 10:13:07 manager INFO: Connected
2022-04-07 10:13:10 sync INFO: Sync aborted
2022-04-07 10:13:10 manager INFO: Paused
2022-04-11 18:06:38 daemon INFO: Starting daemon
2022-04-11 18:06:38 manager INFO: Connected
2022-04-11 18:06:43 sync INFO: Sync aborted
2022-04-11 18:06:43 manager INFO: Paused
2022-04-12 15:03:45 daemon INFO: Starting daemon
2022-04-12 15:03:45 daemon DEBUG: Environment:
{'COLORFGBG': '7;0',
 'COLORTERM': 'truecolor',
 'COMMAND_MODE': 'unix2003',
 'EDITOR': '/usr/local/bin/mate -w',
 'FIG_INTEGRATION_VERSION': '8',
 'FIG_TERM': '1',
 'FIG_TERM_VERSION': '4.3.0',
 'HOME': '/Users/thomasbraun',
 'HOMEBREW_CELLAR': '/opt/homebrew/Cellar',
 'HOMEBREW_PREFIX': '/opt/homebrew',
 'HOMEBREW_REPOSITORY': '/opt/homebrew',
 'INFOPATH': '/opt/homebrew/share/info:',
 'ITERM_PROFILE': 'Default',
 'ITERM_SESSION_ID': 'w0t1p0:E1F08CC4-AFD1-45BB-9E06-CEA2E7B20378',
 'LANG': 'en_US.UTF-8',
 'LC_CTYPE': 'UTF-8',
 'LC_TERMINAL': 'iTerm2',
 'LC_TERMINAL_VERSION': '3.4.15',
 'LESS': '-R',
 'LOGNAME': 'thomasbraun',
 'LSCOLORS': 'Gxfxcxdxbxegedabagacad',
 'MANPATH': '/opt/homebrew/share/man::',
 'PAGER': 'less',
 'PATH': '/opt/homebrew/lib/ruby/gems/3.1.0/bin:/opt/homebrew/opt/ruby/bin:/Users/thomasbraun/bin:/usr/local/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/thomasbraun/.gem/ruby/2.6.0/bin:/Applications/Little '
         'Snitch.app/Contents/Components:/usr/local/share/dotnet:~/.dotnet/tools:/Library/Apple/usr/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/thomasbraun/.local/bin:/Users/thomasbraun/.fig/bin:/Users/thomasbraun/.local/bin:/Users/thomasbraun/.local/bin:/Users/thomasbraun/.cabal/bin:/Users/thomasbraun/.ghcup/bin',
 'PWD': '/Users/thomasbraun',
 'PYTHONDONTWRITEBYTECODE': '1',
 'PYTHONOPTIMIZE': '2',
 'PYTHONPATH': '/Applications/Maestral.app/Contents/Resources/app:/Applications/Maestral.app/Contents/Resources/app_packages',
 'PYTHONUNBUFFERED': '1',
 'SHELL': '/bin/zsh',
 'SHLVL': '2',
 'SSH_AUTH_SOCK': '/private/tmp/com.apple.launchd.U3U3fIs8vH/Listeners',
 'TERM': 'xterm-256color',
 'TERM_PROGRAM': 'iTerm.app',
 'TERM_PROGRAM_VERSION': '3.4.15',
 'TERM_SESSION_ID': 'w0t1p0:E1F08CC4-AFD1-45BB-9E06-CEA2E7B20378',
 'TMPDIR': '/var/folders/_k/c451cfq96dj1h3txk4g2q7gr0000gn/T/',
 'TTY': '/dev/ttys004',
 'USER': 'thomasbraun',
 'XPC_FLAGS': '0x0',
 'XPC_SERVICE_NAME': '0',
 'ZSH': '/Users/thomasbraun/.oh-my-zsh',
 '_': '/Applications/Maestral.app/Contents/MacOS/Maestral',
 '__CFBundleIdentifier': 'com.googlecode.iterm2',
 '__CF_USER_TEXT_ENCODING': '0x1F5:0x0:0x0'}
2022-04-12 15:03:45 daemon DEBUG: Acquired daemon lock: '/Users/thomasbraun/Library/Application Support/maestral/maestral.lock'
2022-04-12 15:03:45 daemon DEBUG: Integrating with CFEventLoop
2022-04-12 15:03:45 daemon DEBUG: Socket path: '/Users/thomasbraun/Library/Application Support/maestral/maestral.sock'
2022-04-12 15:03:45 daemon DEBUG: Creating Pyro daemon
2022-04-12 15:03:45 sync DEBUG: Could not load mignore rules: No such file or directory
2022-04-12 15:03:45 daemon DEBUG: Starting event loop
2022-04-12 15:03:45 manager INFO: Connected
2022-04-12 15:06:10 sync INFO: Sync aborted
2022-04-12 15:06:10 manager INFO: Paused

```

**Comment by samschott** [comment]
Yeah, I can see that it's starting up fine now. Though it does not explain the previous error 😕

The only scenario leading to this error which I can image at the moment is when the app bundle is moved while the app is running. This could result in the certificate file no longer existing where it was initially loaded from.

The fact that Maestral starts at all indicates that `cacert.pem` was successfully located during startup, it should have otherwise resulted in a hard crash.

Since you have ruled that out, I am really out of ideas here. Though I am glad everything is working now!

**Comment by tbraun1551** [comment]
One thing I noticed that seemed a little suspicious is when I went to quit Maestral from Activity Monitor, there was 4 different entries (each connected to different ports). After I restarted the app, there is only 2 entries both connected to port 79 and 80 (although that may change periodically).
Will let you know if I am able to reproduce or find the cause of the earlier issue.
Thanks for all the help :)
